src/emmy/structure.cljc

```diff
@@ -241,56 +249,91 @@
+                (make orientation (mapv #(% a b c d e f g) v) m))
        (-invoke [_ a b c d e f g h]
-                (Structure. orientation (mapv #(% a b c d e f g h) v) m))
+                (make orientation (mapv #(% a b c d e f g h) v) m))
        (-invoke [_ a b c d e f g h i]
-                (Structure. orientation (mapv #(% a b c d e f g h i) v) m))
+                (make orientation (mapv #(% a b c d e f g h i) v) m))
        (-invoke [_ a b c d e f g h i j]
-                (Structure. orientation (mapv #(% a b c d e f g h i j) v) m))
+                (make orientation (mapv #(% a b c d e f g h i j) v) m))
        (-invoke [_ a b c d e f g h i j k]
-                (Structure. orientation (mapv #(% a b c d e f g h i j k) v) m))
+                (make orientation (mapv #(% a b c d e f g h i j k) v) m))
        (-invoke [_ a b c d e f g h i j k l]
-                (Structure. orientation (mapv #(% a b c d e f g h i j k l) v) m))
+                (make orientation (mapv #(% a b c d e f g h i j k l) v) m))
        (-invoke [_ a b c d e f g h i j k l m-arg]
-                (Structure. orientation (mapv #(% a b c d e f g h i j k l m-arg) v) m))
+                (make orientation (mapv #(% a b c d e f g h i j k l m-arg) v) m))
        (-invoke [_ a b c d e f g h i j k l m-arg n]
-                (Structure. orientation (mapv #(% a b c d e f g h i j k l m-arg n) v) m))
+                (make orientation (mapv #(% a b c d e f g h i j k l m-arg n) v) m))
        (-invoke [_ a b c d e f g h i j k l m-arg n o]
-                (Structure. orientation (mapv #(% a b c d e f g h i j k l m-arg n o) v) m))
+                (make orientation (mapv #(% a b c d e f g h i j k l m-arg n o) v) m))
        (-invoke [_ a b c d e f g h i j k l m-arg n o p]
-                (Structure. orientation (mapv #(% a b c d e f g h i j k l m-arg n o p) v) m))
+                (make orientation (mapv #(% a b c d e f g h i j k l m-arg n o p) v) m))
        (-invoke [_ a b c d e f g h i j k l m-arg n o p q]
-                (Structure. orientation (mapv #(% a b c d e f g h i j k l m-arg n o p q) v) m))
+                (make orientation (mapv #(% a b c d e f g h i j k l m-arg n o p q) v) m))
        (-invoke [_ a b c d e f g h i j k l m-arg n o p q r]
-                (Structure. orientation (mapv #(% a b c d e f g h i j k l m-arg n o p q r) v) m))
+                (make orientation (mapv #(% a b c d e f g h i j k l m-arg n o p q r) v) m))
        (-invoke [_ a b c d e f g h i j k l m-arg n o p q r s]
-                (Structure. orientation (mapv #(% a b c d e f g h i j k l m-arg n o p q r s) v) m))
+                (make orientation (mapv #(% a b c d e f g h i j k l m-arg n o p q r s) v) m))
        (-invoke [_ a b c d e f g h i j k l m-arg n o p q r s t]
-                (Structure. orientation (mapv #(% a b c d e f g h i j k l m-arg n o p q r s t) v) m))
+                (make orientation (mapv #(% a b c d e f g h i j k l m-arg n o p q r s t) v) m))
```

Comment:
I really want a macro that will auto-expand out these IFn patterns in our deftypes! We'll get one at some point

Reply:
this wasn't necessary after all so it's gone anyway.